pipelines/visitQAEllipPlots.yaml

```diff
@@ -270,6 +270,29 @@ tasks:
         from lsst.analysis.drp.calcFunctors import CalcShapeSize
         from lsst.pipe.tasks.dataFrameActions import SubtractColumns
 
+  plot_shapeSizeResidsDiff_scatter_visit:
+    class: lsst.analysis.drp.scatterPlotVisit.ScatterPlotVisitTask
+    config:
+      axisActions.yAction: FractionalResidualColumns
+      axisActions.yAction.colA: CalcShapeSize
+      axisActions.yAction.colA.colXx: "ixx"
+      axisActions.yAction.colA.colYy: "iyy"
+      axisActions.yAction.colB: CalcShapeSize
+      axisActions.yAction.colB.colXx: "ixxPSF"
+      axisActions.yAction.colB.colYy: "iyyPSF"
+      connections.plotName: shapeFractionDiff_sizePSF_scatter
+      selectorActions.flagSelector: VisitPlotFlagSelector
+      selectorActions.SnSelector: SnSelector
+      selectorActions.SnSelector.bands: [""]
+      selectorActions.SnSelector.fluxType: "psfFlux"
+      selectorActions.SnSelector.threshold: 10
+      axisLabels: {"x": "PSF Magnitude (mag)", "mag": "PSF Magnitude (mag)",
+                   "y": "Fractional shape difference (ixx - ixxPSF)"}
```

Comment:
Just spoke with you about this label only referencing `ixx`, but realize that this will be addessed in DM-33971, so that's okay.